Makefile

```
@@ -1,5 +1,10 @@
 install:
-	sudo ln -fs $(shell pwd)/create-container.sh /usr/sbin/devenv
+	cp create-container.sh /usr/sbin/devenv
+	cp config /etc/devenv
+
+uninstall:
+	rm create-container.sh /usr/sbin/devenv
+	rm config /etc/devenv
```

Shouldn't this be something like:
```
  rm /usr/sbin/devenv/create-container.sh
  rm /etc/devenv/config
```

---

Not exactly. To be coherent with `install`, it should read:
```suggestion
uninstall:
	rm /usr/sbin/devenv
	rm /etc/devenv
```

Thank you for reviewing it!